openshift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This will produce and image to be used in OpenShift
+# Build should be triggered from the repo root like:
+# docker build -f Dockerfile --tag 172.30.1.1:5000/myproject/resultsdb-updater:latest --build-arg resultsdb_updater_rpm=resultsdb-updater-2.4.0-1.el7eng.noarch.rpm .
```

Comment:
Can you just pull the latest resultsdb-updater rpm from the repo, instead of having to pass it as a build arg?